server.js

```diff
@@ -1,34 +1,34 @@
 // TASK 1: Setting up server
 
 
-const http = require('http');
+// const http = require('http');
 
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');

```

Comment:
The 'http' module is used in this code snippet but it is not imported. It needs to be imported at the beginning of the file using: const http = require('http');